### DISCUSSION TITLE
MonitorControl 4.20 Build 7048 crashes on Sequoia - UPDATED BUILD AVAILABLE!

### DISCUSSION THREAD

**Discussion by fKing** [MonitorControl/MonitorControl#1572] [Q&A] [answered]
it's only Dev Beta 1 of the OS, but monitorControl quits when i try to change brightness on my M2 MBA and a Dell monitor. just reporting it! (and hoping for an updated app soon)

**Comment by waydabber** [comment, marked as answer]
**An updated (unofficial) build with Sequoia compatibility [can be downloaded here](https://github.com/MonitorControl/MonitorControl/issues/1570)!**

**Comment by fKing** [comment]
working! thanks so much for this..

**Comment by wclyffe4** [comment]
Anyone using the latest Monitor Control 4.31 (Beta) w/ a MacBookPro M2 and a 27" Apple Cinema Display (w/Answin USB-C to Mini DisplayPort adapter) on OS Sequoia?

**Comment by waydabber** [comment]
Hi all, posted an official v4.3.2 that should fix all issues:

- Sequoia compatibility
- No DDC control on third party displays in HDR mode
- Custom keyboard shortcuts not working
- Dock freezing on exit

Obviously the app could be improved in many ways still, the release was meant to simply make the app work on Sequoia as we all got used to it over the years. For more features, there is [BetterDisplay](https://betterdisplay.pro). :)
